### ISSUE TITLE
Custom close button

### ISSUE THREAD

**Issue by ritz078** [sniphpet/react-toastify#5]
Will be nice if I can use close button from my React component. 

API may look like

```js
toast(close => <CustomToast onClose={close} text='hello' type='info'/>, options)
```

**Comment by sniphpet** [comment]
Forgot to say good idea :+1:

**Comment by sniphpet** [comment]
@ritz078 This feature will be available for the release 1.3.0. I will probably publish it tomorrow on npm.

**Comment by sniphpet** [comment]
@ritz078 I published the v1.3.0
